unity-renderer/Assets/Scripts/MainScripts/DCL/Environment/Factories/ServiceLocatorFactory/ServiceLocatorFactory.cs

```diff
@@ -0,0 +1,43 @@
+﻿using DCL.Controllers;
+using DCL.Rendering;
+using UnityEngine;
+
+namespace DCL
+{
+    public static class ServiceLocatorFactory
```

Comment:
I see this as an util rather than a factory, even a method in the `ServiceLocator` itself.